pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -540,20 +550,27 @@
    "outputs": [],
    "source": [
     "# pipeline-api\n",
-    "def pipeline_api(file, filename='', response_type=\"application/json\"):\n",
+    "\n",
+    "def pipeline_api(file, filename='', file_content_type=None, response_type=\"application/json\"):\n",
     "    # NOTE(robinson) - This is a hacky solution due to\n",
     "    # limitations in the SpooledTemporaryFile wrapper.\n",
-    "    # Specifically, it does't have a `seekable` attribute,\n",
+    "    # Specifically, it doesn't have a `seekable` attribute,\n",
     "    # which is required for .pptx and .docx. See below\n",
     "    # the link below\n",
     "    # ref: https://stackoverflow.com/questions/47160211\n",
     "    # /why-doesnt-tempfile-spooledtemporaryfile-implement-readable-writable-seekable\n",
     "    with tempfile.TemporaryDirectory() as tmpdir:\n",
-    "        _filename = os.path.join(tmpdir, filename.split('/')[-1])\n",
-    "        with open(_filename, \"wb\") as f:\n",
+    "        file_filename = os.path.join(tmpdir, filename.split('/')[-1])\n",
```

Comment:
this is causing the client's filename that was specified in their POST, e.g. `'sample-docs/alert.eml'` to get replaced with `alert.eml`, which then means the `metadata["filename"]` in elements later gets set to `alert.eml`, which is incorrect. we should honor the metadata `filename` the user included in their request.